shared/templates/cond_package/oval.template

```diff
@@ -0,0 +1,20 @@
+<def-group>
+  <definition class="inventory" id="{{{ _RULE_ID }}}"
```

Comment:
The `_RULE_ID` here probably isn't rule ID but a "platform name". Can we name the variables better?